docs/source/api/APIRef.rst

```diff
@@ -236,7 +236,7 @@ Searching for datasets locations
             import requests
             response = requests.get(
                 "http://api.freva.example/api/databrowser/data_search/freva/file",
-                pramas={"product": "EUR-11", "fs_type": "swift"}
```

Comment:
This get's auto overridden, if you wanted to change something then you'll have to do it here: https://github.com/FREVA-CLINT/freva-nextgen/blob/main/docs/source/databrowser/APIRef.rst